src/components/CreateExport/CreateExportModal.tsx

```diff
@@ -62,6 +63,12 @@ export function CreateExportModal({
     );
   }
 
+  function handleClose() {
+    setIncludedIssueTypes([]);
+    setIncludedIssueStatus([]);
+    setOpened(false);
+  }
```

Comment:
```suggestion
  function closeModal() {
    setIncludedIssueTypes([]);
    setIncludedIssueStatus([]);
    setOpened(false);
  }
```
When you add a close function, use it everywhere it can be used! E.g. line 79 - 83 in `Modal.onClose`.